src/Textarea.js

```diff
@@ -55,7 +55,9 @@ const Textarea = ({
         placeholder={placeholder}
         id={id}
         value={value}
+        defaultValue={defaultValue}
         ref={textareaRef}
+        onChange={onChange}
```

Comment:
nice!, since we're using `...rest` how about delete L21 and L23 and that's it?

Reply:
I prefer the explicit list of parameters. `...rest` is for HTMLTextareaElement attributes, that have no special meaning in react or materialize.

Reply:
fair enough